glInfoExample does not compile

What the title says, throws lots of `undefined reference to` errors. `install_dependencies` has been executed.  Originally I thought this had to do with virtual machines and less common linux distros I was playing around when setting up my Jenkins server. But, I have just confirmed this also happens on my Ubuntu12.04 64bit machine. anybody know what`s wrong here? @arturoc ?
I have milestoned this for 0072 cause I think we shouldn't ship broken examples.

Theory: I got libglew1.5 and libglew1.6 installed, but only libglew1.5-dev, not libglew1.6-dev. could that be it? If I want to install libglew1.6-dev it wants to remove libglew1.5-dev.

console output:

```
-----------------------------------------------------------------
building  + glInfoExample
compiling x86_64 object for:  src/glewOutput.cpp
mkdir -p obj/x86_64Debug/src
g++ -c  -pthread -I/usr/include/gtk-2.0 -I/usr/lib/x86_64-linux-gnu/gtk-2.0/include -I/usr/include/atk-1.0 -I/usr/include/cairo -I/usr/include/gdk-pixbuf-2.0 -I/usr/include/pango-1.0 -I/usr/include/gio-unix-2.0/ -I/usr/include/glib-2.0 -I/usr/lib/x86_64-linux-gnu/glib-2.0/include -I/usr/include/pixman-1 -I/usr/include/freetype2 -I/usr/include/libpng12   -DOF_USING_GTK -DOF_USING_MPG123 -Wall -fexceptions -I. -I../../../libs/glu/include -pthread -I/usr/include/gstreamer-0.10 -I/usr/include/glib-2.0 -I/usr/lib/x86_64-linux-gnu/glib-2.0/include -I/usr/include/libxml2 -I/usr/include/cairo -I/usr/include/pixman-1 -I/usr/include/freetype2 -I/usr/include/libpng12   -I../../../libs/FreeImage/include -I../../../libs/assimp/include -I../../../libs/assimp/include/Compiler -I../../../libs/cairo/include -I../../../libs/cairo/include/cairo -I../../../libs/cairo/include/libpng15 -I../../../libs/cairo/include/pixman-1 -I../../../libs/fmodex/include -I../../../libs/freetype/include -I../../../libs/freetype/include/freetype2 -I../../../libs/freetype/include/freetype2/freetype -I../../../libs/freetype/include/freetype2/freetype/config -I../../../libs/freetype/include/freetype2/freetype/internal -I../../../libs/freetype/include/freetype2/freetype/internal/services -I../../../libs/glew/include -I../../../libs/glew/include/GL -I../../../libs/kiss/include -I../../../libs/portaudio/include -I../../../libs/rtAudio/include -I../../../libs/tess2/include -I../../../libs/videoInput/include -I../../../libs/poco/include -I../../../libs/openFrameworks/ -I../../../libs/openFrameworks/.settings -I../../../libs/openFrameworks/3d -I../../../libs/openFrameworks/app -I../../../libs/openFrameworks/communication -I../../../libs/openFrameworks/events -I../../../libs/openFrameworks/gl -I../../../libs/openFrameworks/graphics -I../../../libs/openFrameworks/math -I../../../libs/openFrameworks/sound -I../../../libs/openFrameworks/types -I../../../libs/openFrameworks/utils -I../../../libs/openFrameworks/video -g3   -MMD -MP -MFobj/x86_64Debug/src/glewOutput.d -MTobj/x86_64Debug/src/glewOutput.o -oobj/x86_64Debug/src/glewOutput.o -c src/glewOutput.cpp
compiling x86_64 object for:  src/main.cpp
mkdir -p obj/x86_64Debug/src
g++ -c  -pthread -I/usr/include/gtk-2.0 -I/usr/lib/x86_64-linux-gnu/gtk-2.0/include -I/usr/include/atk-1.0 -I/usr/include/cairo -I/usr/include/gdk-pixbuf-2.0 -I/usr/include/pango-1.0 -I/usr/include/gio-unix-2.0/ -I/usr/include/glib-2.0 -I/usr/lib/x86_64-linux-gnu/glib-2.0/include -I/usr/include/pixman-1 -I/usr/include/freetype2 -I/usr/include/libpng12   -DOF_USING_GTK -DOF_USING_MPG123 -Wall -fexceptions -I. -I../../../libs/glu/include -pthread -I/usr/include/gstreamer-0.10 -I/usr/include/glib-2.0 -I/usr/lib/x86_64-linux-gnu/glib-2.0/include -I/usr/include/libxml2 -I/usr/include/cairo -I/usr/include/pixman-1 -I/usr/include/freetype2 -I/usr/include/libpng12   -I../../../libs/FreeImage/include -I../../../libs/assimp/include -I../../../libs/assimp/include/Compiler -I../../../libs/cairo/include -I../../../libs/cairo/include/cairo -I../../../libs/cairo/include/libpng15 -I../../../libs/cairo/include/pixman-1 -I../../../libs/fmodex/include -I../../../libs/freetype/include -I../../../libs/freetype/include/freetype2 -I../../../libs/freetype/include/freetype2/freetype -I../../../libs/freetype/include/freetype2/freetype/config -I../../../libs/freetype/include/freetype2/freetype/internal -I../../../libs/freetype/include/freetype2/freetype/internal/services -I../../../libs/glew/include -I../../../libs/glew/include/GL -I../../../libs/kiss/include -I../../../libs/portaudio/include -I../../../libs/rtAudio/include -I../../../libs/tess2/include -I../../../libs/videoInput/include -I../../../libs/poco/include -I../../../libs/openFrameworks/ -I../../../libs/openFrameworks/.settings -I../../../libs/openFrameworks/3d -I../../../libs/openFrameworks/app -I../../../libs/openFrameworks/communication -I../../../libs/openFrameworks/events -I../../../libs/openFrameworks/gl -I../../../libs/openFrameworks/graphics -I../../../libs/openFrameworks/math -I../../../libs/openFrameworks/sound -I../../../libs/openFrameworks/types -I../../../libs/openFrameworks/utils -I../../../libs/openFrameworks/video -g3   -MMD -MP -MFobj/x86_64Debug/src/main.d -MTobj/x86_64Debug/src/main.o -oobj/x86_64Debug/src/main.o -c src/main.cpp
compiling x86_64 object for:  src/testApp.cpp
mkdir -p obj/x86_64Debug/src
g++ -c  -pthread -I/usr/include/gtk-2.0 -I/usr/lib/x86_64-linux-gnu/gtk-2.0/include -I/usr/include/atk-1.0 -I/usr/include/cairo -I/usr/include/gdk-pixbuf-2.0 -I/usr/include/pango-1.0 -I/usr/include/gio-unix-2.0/ -I/usr/include/glib-2.0 -I/usr/lib/x86_64-linux-gnu/glib-2.0/include -I/usr/include/pixman-1 -I/usr/include/freetype2 -I/usr/include/libpng12   -DOF_USING_GTK -DOF_USING_MPG123 -Wall -fexceptions -I. -I../../../libs/glu/include -pthread -I/usr/include/gstreamer-0.10 -I/usr/include/glib-2.0 -I/usr/lib/x86_64-linux-gnu/glib-2.0/include -I/usr/include/libxml2 -I/usr/include/cairo -I/usr/include/pixman-1 -I/usr/include/freetype2 -I/usr/include/libpng12   -I../../../libs/FreeImage/include -I../../../libs/assimp/include -I../../../libs/assimp/include/Compiler -I../../../libs/cairo/include -I../../../libs/cairo/include/cairo -I../../../libs/cairo/include/libpng15 -I../../../libs/cairo/include/pixman-1 -I../../../libs/fmodex/include -I../../../libs/freetype/include -I../../../libs/freetype/include/freetype2 -I../../../libs/freetype/include/freetype2/freetype -I../../../libs/freetype/include/freetype2/freetype/config -I../../../libs/freetype/include/freetype2/freetype/internal -I../../../libs/freetype/include/freetype2/freetype/internal/services -I../../../libs/glew/include -I../../../libs/glew/include/GL -I../../../libs/kiss/include -I../../../libs/portaudio/include -I../../../libs/rtAudio/include -I../../../libs/tess2/include -I../../../libs/videoInput/include -I../../../libs/poco/include -I../../../libs/openFrameworks/ -I../../../libs/openFrameworks/.settings -I../../../libs/openFrameworks/3d -I../../../libs/openFrameworks/app -I../../../libs/openFrameworks/communication -I../../../libs/openFrameworks/events -I../../../libs/openFrameworks/gl -I../../../libs/openFrameworks/graphics -I../../../libs/openFrameworks/math -I../../../libs/openFrameworks/sound -I../../../libs/openFrameworks/types -I../../../libs/openFrameworks/utils -I../../../libs/openFrameworks/video -g3   -MMD -MP -MFobj/x86_64Debug/src/testApp.d -MTobj/x86_64Debug/src/testApp.o -oobj/x86_64Debug/src/testApp.o -c src/testApp.cpp
src/testApp.cpp: In function 'void print_extension_list(char*)':
src/testApp.cpp:54:24: warning: format not a string literal and no format arguments [-Wformat-security]
src/testApp.cpp:64:36: warning: format not a string literal and no format arguments [-Wformat-security]
src/testApp.cpp: In function 'void printGLInfo()':
src/testApp.cpp:167:11: warning: unused variable 'extensions' [-Wunused-variable]
src/testApp.cpp:168:11: warning: unused variable 'glutVersion' [-Wunused-variable]
linking x86_64 bin/glInfoExample_debug linux64
mkdir -p bin
g++ -o bin/glInfoExample_debug obj/x86_64Debug/src/glewOutput.o obj/x86_64Debug/src/main.o obj/x86_64Debug/src/testApp.o   -Wl,-rpath=./libs  ../../../libs/openFrameworksCompiled/lib/linux64/libopenFrameworksDebug.a   ../../../libs/poco/lib/linux64/libPocoNet.a ../../../libs/poco/lib/linux64/libPocoXML.a ../../../libs/poco/lib/linux64/libPocoUtil.a ../../../libs/poco/lib/linux64/libPocoFoundation.a ../../../libs/freetype/lib/linux64/libfreetype.a ../../../libs/kiss/lib/linux64/libkiss.a ../../../libs/portaudio/lib/linux64/libportaudio.a ../../../libs/rtAudio/lib/linux64/libRtAudio.a ../../../libs/tess2/lib/linux64/libtess2.a -L../../../libs/fmodex/lib/linux64 -L../../../libs/freetype/lib/linux64 -L../../../libs/kiss/lib/linux64 -L../../../libs/openFrameworksCompiled/lib/linux64 -L../../../libs/poco/lib/linux64 -L../../../libs/portaudio/lib/linux64 -L../../../libs/rtAudio/lib/linux64 -L../../../libs/tess2/lib/linux64 -lfmodex -lgtk-x11-2.0 -lgdk-x11-2.0 -latk-1.0 -lgio-2.0 -lpangoft2-1.0 -lpangocairo-1.0 -lgdk_pixbuf-2.0 -lcairo -lpango-1.0 -lfreetype -lfontconfig -lgobject-2.0 -lglib-2.0   -lmpg123 -pthread -ljack -lpthread -lGLEW -lGLU -lGL -lgstvideo-0.10 -lgstapp-0.10 -lgstbase-0.10 -lgstreamer-0.10 -lgobject-2.0 -lgmodule-2.0 -lgthread-2.0 -lxml2 -lglib-2.0 -ludev -lrt -lcairo -lz   -lglut -lGL -lasound -lopenal -lsndfile -lvorbis -lFLAC -logg -lfreeimage
obj/x86_64Debug/src/glewOutput.o: In function `glewInfo_GL_VERSION_1_2_1()':
/media/windata/Visuals/Coding/openFrameworks/examples/gl/glInfoExample/src/glewOutput.cpp:57: undefined reference to `__GLEW_VERSION_1_2_1'
/media/windata/Visuals/Coding/openFrameworks/examples/gl/glInfoExample/src/glewOutput.cpp:57: undefined reference to `__GLEW_VERSION_1_2_1'
/media/windata/Visuals/Coding/openFrameworks/examples/gl/glInfoExample/src/glewOutput.cpp:57: undefined reference to `__GLEW_VERSION_1_2_1'
obj/x86_64Debug/src/glewOutput.o: In function `glewInfo_GL_VERSION_3_3()':
/media/windata/Visuals/Coding/openFrameworks/examples/gl/glInfoExample/src/glewOutput.cpp:419: undefined reference to `__GLEW_VERSION_3_3'
/media/windata/Visuals/Coding/openFrameworks/examples/gl/glInfoExample/src/glewOutput.cpp:419: undefined reference to `__GLEW_VERSION_3_3'
/media/windata/Visuals/Coding/openFrameworks/examples/gl/glInfoExample/src/glewOutput.cpp:419: undefined reference to `__GLEW_VERSION_3_3'
/media/windata/Visuals/Coding/openFrameworks/examples/gl/glInfoExample/src/glewOutput.cpp:421: undefined reference to `__glewVertexAttribDivisor'
obj/x86_64Debug/src/glewOutput.o: In function `glewInfo_GL_VERSION_4_0()':
/media/windata/Visuals/Coding/openFrameworks/examples/gl/glInfoExample/src/glewOutput.cpp:430: undefined reference to `__GLEW_VERSION_4_0'
/media/windata/Visuals/Coding/openFrameworks/examples/gl/glInfoExample/src/glewOutput.cpp:430: undefined reference to `__GLEW_VERSION_4_0'
/media/windata/Visuals/Coding/openFrameworks/examples/gl/glInfoExample/src/glewOutput.cpp:430: undefined reference to `__GLEW_VERSION_4_0'
/media/windata/Visuals/Coding/openFrameworks/examples/gl/glInfoExample/src/glewOutput.cpp:432: undefined reference to `__glewBlendEquationSeparatei'
/media/windata/Visuals/Coding/openFrameworks/examples/gl/glInfoExample/src/glewOutput.cpp:433: undefined reference to `__glewBlendEquationi'
/media/windata/Visuals/Coding/openFrameworks/examples/gl/glInfoExample/src/glewOutput.cpp:434: undefined reference to `__glewBlendFuncSeparatei'
/media/windata/Visuals/Coding/openFrameworks/examples/gl/glInfoExample/src/glewOutput.cpp:435: undefined reference to `__glewBlendFunci'
/media/windata/Visuals/Coding/openFrameworks/examples/gl/glInfoExample/src/glewOutput.cpp:436: undefined reference to `__glewMinSampleShading'
obj/x86_64Debug/src/glewOutput.o: In function `glewInfo_GL_VERSION_4_1()':
/media/windata/Visuals/Coding/openFrameworks/examples/gl/glInfoExample/src/glewOutput.cpp:445: undefined reference to `__GLEW_VERSION_4_1'
/media/windata/Visuals/Coding/openFrameworks/examples/gl/glInfoExample/src/glewOutput.cpp:445: undefined reference to `__GLEW_VERSION_4_1'
/media/windata/Visuals/Coding/openFrameworks/examples/gl/glInfoExample/src/glewOutput.cpp:445: undefined reference to `__GLEW_VERSION_4_1'
collect2: ld returned 1 exit status
make: *** [bin/glInfoExample_debug] Error 1
error compiling glInfoExample
```

--- a/scripts/linux/debian/install_dependencies.sh
+++ b/scripts/linux/debian/install_dependencies.sh
@@ -1,5 +1,5 @@
 apt-get update
-apt-get install libgstreamer0.10-dev libgstreamer-plugins-base0.10-dev libavcodec-dev libavformat-dev libavutil-dev libswscale-dev freeglut3-dev libasound2-dev libxmu-dev libxxf86vm-dev g++ libgl1-mesa-dev libglu1-mesa-dev libraw1394-dev libudev-dev libdrm-dev gstreamer0.10-ffmpeg libglew1.5-dev libopenal-dev libsndfile-dev libfreeimage-dev libcairo2-dev libgtk2.0-dev libjack-dev python-lxml python-argparse
+apt-get install libgstreamer0.10-dev libgstreamer-plugins-base0.10-dev libavcodec-dev libavformat-dev libavutil-dev libswscale-dev freeglut3-dev libasound2-dev libxmu-dev libxxf86vm-dev g++ libgl1-mesa-dev libglu1-mesa-dev libraw1394-dev libudev-dev libdrm-dev gstreamer0.10-ffmpeg libglew-dev libopenal-dev libsndfile-dev libfreeimage-dev libcairo2-dev libgtk2.0-dev libjack-dev python-lxml python-argparse
 ARCH=$(uname -m)
 if [ "$ARCH" = "x86_64" ]; then
         LIBSPATH=linux64

--- a/scripts/linux/ubuntu/install_dependencies.sh
+++ b/scripts/linux/ubuntu/install_dependencies.sh
@@ -1,5 +1,5 @@
 apt-get update
-apt-get install libgstreamer0.10-dev libgstreamer-plugins-base0.10-dev libavcodec-dev libavformat-dev libavutil-dev libswscale-dev freeglut3-dev libasound2-dev libxmu-dev libxxf86vm-dev g++ libgl1-mesa-dev libglu1-mesa-dev libraw1394-dev libudev-dev libdrm-dev gstreamer0.10-ffmpeg libglew1.5-dev libopenal-dev libsndfile-dev libfreeimage-dev libcairo2-dev libgtk2.0-dev libjack0 libjack-dev python-lxml python-argparse
+apt-get install libgstreamer0.10-dev libgstreamer-plugins-base0.10-dev libavcodec-dev libavformat-dev libavutil-dev libswscale-dev freeglut3-dev libasound2-dev libxmu-dev libxxf86vm-dev g++ libgl1-mesa-dev libglu1-mesa-dev libraw1394-dev libudev-dev libdrm-dev gstreamer0.10-ffmpeg libglew-dev libopenal-dev libsndfile-dev libfreeimage-dev libcairo2-dev libgtk2.0-dev libjack0 libjack-dev python-lxml python-argparse
 ARCH=$(uname -m)
 if [ "$ARCH" = "x86_64" ]; then
 	LIBSPATH=linux64

